lib/QuipProcessor.js

```diff
@@ -295,8 +295,26 @@ class QuipProcessor {
                 dateStr = updatedDate === createdDate? createdDate : `${createdDate} (Updated: ${updatedDate})`;
                 dateStr = `, ${dateStr}`;
             }
+
+            let highlightedSections = "";
+            if (
+                message.annotation &&
```

Comment:
This formatting is due to [prettier](https://prettier.io/), but I did notice that it doesn't seem like there's a line length limit being enforced in this codebase, so let me know if you'd prefer this to be consolidated onto one line. Same goes for the other code in this patch!